templates/partials/livestream-notification.html

```diff
@@ -0,0 +1,19 @@
+{% if is_stream_live == 0 %}
+<div class="col-12">
+    <div class="p-notification--information p-snapcraft-nps-banner">
+        <p class="p-notification__response">
+        <span class="p-notification__status">LIVE:</span>
+        <a href="https://www.youtube.com/watch?v=BEp_l2oUcD8&amp;ab_channel=snapcraft" target="_blank" class="p-link--external">Watch our Youtube Live Stream on building snaps</a> this Wednesday 6 Feb 2019 @ 20:00 GMT
+        </p>
+    </div>
+</div>
+{% elif is_stream_live == 1 %}
+<div class="col-12">
+    <div class="p-notification--information p-snapcraft-nps-banner">
+        <p class="p-notification__response">
+        <span class="p-notification__status">LIVE:</span>
+        <a href="https://www.youtube.com/watch?v=BEp_l2oUcD8&amp;ab_channel=snapcraft" target="_blank" class="p-link--external">Watch our Youtube Live Stream on building snaps</a> is now!
```

Comment:
"Watch our Youtube Live Stream on building snaps is now!" doesn't sound right. I guess we could drop "is"?

Reply:
lol I didnt' read

Reply:
![https://media.giphy.com/media/c793Kq1vwkss8/giphy.gif](https://media.giphy.com/media/c793Kq1vwkss8/giphy.gif)

Reply:
better like this?
![image](https://user-images.githubusercontent.com/2707508/52284317-15bdc880-295c-11e9-95d3-d20376a71a60.png)

Reply:
👍